tests/suites/test_suite_platform.data

```diff
@@ -10,3 +10,6 @@ time_delay_milliseconds:1000
 
 Time: delay seconds
 time_delay_seconds:1
+
+Check mbedtls_calloc zeroization
+check_mbedtls_calloc_zeroization:
```

Comment:
Arguably out of scope, but we should test `mbedtls_calloc` with a variety of parameters. In particular (0, x) and (x, 0).